### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mattcollier**
@dlongley is this module supposed to be a thing?  We used it for a POC, then apparently it was completely dropped on the floor.

### CODE REVIEW

**Comment by dlongley** [comment]
Yes, it is supposed to be used, just haven't gotten around to it. I need to spend some time reviewing the API given lessons learned before we can pull it in.

**Comment by aljones15** [comment]
@mattcollier is this ready for master? If not I can use this branch.

File: test/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "http://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-key-http/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-key-http",
```
Comment (mattcollier) [inline test/package.json]: trying out multi-line comments.  some of this stuff is not accurate (bedrock-key), but we don't need this stuff in the test package file, this file should have bare minimum required.

File: test/mocha/mock.data.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "verifiableCredential": [],
+};
+
+const privateDidDocuments = mock.privateDidDocuments = {};
+
+privateDidDocuments.alpha = {
+  "@context": didContexts,
+  "id": "did:v1:test:nym:z279yHL6HsxRzCPU78DAWgZVieb8xPK1mJKJBbP8T2CezuFY",
+  // FIXME should generate a new key for this.
+  "assertionMethod": [
```
Comment (dlongley) [inline test/mocha/mock.data.js]: If this didn't end up getting used in the stuff we did @aljones15, let's remove it.

Comment (aljones15) [reply test/mocha/mock.data.js]: Addressed here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/649b8d87e76e10658c6c700878807b883d8587e1

File: test/package.json (hunk 1 above)
Comment (aljones15) [reply test/package.json]: I cut it down and removed `bedrock-key` here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/649b8d87e76e10658c6c700878807b883d8587e1

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2019 Digital Bazaar, Inc.
```
Comment (mattcollier) [inline LICENSE.md]:
```suggestion
Copyright (c) 2011-2020 Digital Bazaar, Inc.
```

File: test/test.config.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) 2012-2019 Digital Bazaar, Inc. All rights reserved.
```
Comment (mattcollier) [inline test/test.config.js]: sorry, still more in ancillary files.

Comment (aljones15) [reply test/test.config.js]: Addressed: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/ba56da9231789067b9e3b3c26ae560a452a64c85

File: LICENSE.md (hunk 3 above)
Comment (aljones15) [reply LICENSE.md]: Addressed (the diff was outdated) https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/ba56da9231789067b9e3b3c26ae560a452a64c85
